website_sale_qty/static/tests/js/website_sale_qty.tour.js

```diff
@@ -0,0 +1,105 @@
+/**
+*    Copyright 2016 LasLabs Inc.
+*    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+**/
+
+odoo.define('website_sale.tour', function(require) {
+    'use strict';
+
+    var tour = require('web.Tour');
+    var _t = require('web.core')._t;
+
+    tour.register({
```

Comment:
See [this diff](https://github.com/OCA/website/pull/329/files#diff-303eb7ed9721d5e1a330c9897074c32dR5) for upgrading to v10